webapi/ticketstatus.go

```diff
@@ -34,12 +34,24 @@ func ticketStatus(c *gin.Context) {
 		return
 	}
 
+	// Get altSignAddress from database
+	altSignAddrData, err := db.AltSignAddrData(ticket.Hash)
+	if err != nil {
+		log.Errorf("db.AltSignAddrData error (ticketHash=%s): %v", ticket.Hash, err)
```

Comment:
I think we probably want to return an error to the client if a database error happens here.